### ISSUE TITLE
Multifox No Longer Offers to "Open in a new identity profile" with multipe Google Products/tabs?

### ISSUE THREAD

**Issue by bondbeau** [hultmann/multifox#94]
I'd started another thread a long time ago saying that Multifox breaks capitalone.com. Now that was eventually fixed. Thanks much Jeferson.

But in the meantime, in Firefox (now 17.0.1) on Windows 7 Ultimate 64-bit, I'd noticed that Multifox didn't show up in the search bar anymore for Google properties...so no option to choose "Open in a new identity profile" for me...and I have no clue why.

Looked to see if I was running the latest Multifox and I believe I am. But if I am logged in to Gmail (and I almost always am), Google follows me right through to any open tab in Firefox if I visit one of their products, EVEN if I don't have an account for that product.

So in short, where Multifox used to allow me to circumvent Google's Evil Eye, it doesn't seem to offer me that option now. As a workaround, I've restarted using another browser (or two) to keep from being logged (which is what I was doing _before_ Multifox).

Please advise? And thank you!

**Comment by hultmann** [comment]
The extension formerly known as Multifox 2 beta is currently AllAccounts. Try it: http://br.mozdev.org/multifox/all.html#allaccounts

AllAccounts will protect you from tracking, while Multifox doesn't.
